src/zkevm_specs/evm/execution/callop.py

```diff
@@ -16,18 +16,22 @@
 from ..precompiled import PrecompiledAddress
 
 
-def call_staticcall(instruction: Instruction):
+def callop(instruction: Instruction):
     opcode = instruction.opcode_lookup(True)
-    is_call, _ = instruction.pair_select(opcode, Opcode.CALL, Opcode.STATICCALL)
+    is_call, is_delegatecall, is_staticcall = instruction.multiple_select(
+        opcode, (Opcode.CALL, Opcode.DELEGATECALL, Opcode.STATICCALL)
+    )
     instruction.responsible_opcode_lookup(opcode)
 
     callee_call_id = instruction.curr.rw_counter
 
     tx_id = instruction.call_context_lookup(CallContextFieldTag.TxId)
     reversion_info = instruction.reversion_info()
     caller_address = instruction.call_context_lookup(CallContextFieldTag.CalleeAddress)
+    parent_caller_address = instruction.call_context_lookup(CallContextFieldTag.CallerAddress)
+    parent_value = instruction.call_context_lookup(CallContextFieldTag.Value)
```

Comment:
`parent_call_value` name better ?

Reply:
Renamed `parent_value` to `parent_call_value`.